proposals/spi-proposal.md

```diff
@@ -32,11 +34,10 @@ There are 3 ways to send and receive data from the SPI slave device:
 3. `SPI_READ` Only read data, writing `0` for each word to be read.
 
 A `SPI_REPLY` is used to send requested data back to the client application when
-either a `SPI_TRANSFER` mode or `SPI_READ` message is sent.
+either a `SPI_TRANSFER` mode or `SPI_READ` command is sent.
 
 A `SPI_END` command disables the SPI bus for the specified channel.
 
-
 ### SPI_BEGIN
```

Comment:
SPI_BEGIN may not be necessary for Arduino, Linux. It could simply be part of the initialization sequence.

Reply:
SPI_BEGIN is not necessary on Linux if character special device files as described [here](https://www.kernel.org/doc/Documentation/spi/spidev) are being used to access SPI devices. Having it as part of the initialization sequence sounds like a good idea as it may make sense on some systems.